### ISSUE TITLE
Support Laravel 5.8

### ISSUE THREAD

**Issue by StanBarrows** [laravel-notification-channels/facebook-poster#23]
```
 - Installation request for laravel-notification-channels/facebook-poster ^1.4 -> satisfiable by laravel-notification-channels/facebook-poster[1.4.0].
    - Conclusion: remove laravel/framework v5.8.2
    - Conclusion: don't install laravel/framework v5.8.2
```

**Comment by lightwalkernet** [comment]
would be great to get this working on 5.8. Happy to run a Shift on it if that would be helpful?

**Comment by alexisrazok** [comment]
I really hope this support will be available soon

**Comment by itwaree** [comment]
Error validating access token: The session is invalid because the user logged out.

**Comment by itwaree** [comment]
i am getting error ==> (#200) If posting to a group, requires app being installed in the group, and \ either publish_to_groups permission with user token, or both manage_pages \ and publish_pages permission with page token; If posting to a page, \ requires both manage_pages and publish_pages as an admin with \ sufficient administrative permission

**Comment by atymic** [comment]
Closed in favour of #30
